main_routers/characters_router.py

```
@@ -515,28 +515,31 @@ async def update_catgirl_l2d(name: str, request: Request):
                 if 'idle_animation' in data:
-                    if idle_animation is None or idle_animation == '':
-                        set_reserved(characters['猫娘'][name], 'avatar', 'vrm', 'idle_animation', None)
+                    if idle_animation is None or idle_animation == '' or idle_animation == []:
+                        set_reserved(characters['猫娘'][name], 'avatar', 'vrm', 'idle_animation', [])
                     else:
-                        idle_animation_str = str(idle_animation).strip()
-                        if '://' in idle_animation_str or idle_animation_str.startswith('data:'):
-                            return JSONResponse(content={'success': False, 'error': '待机动作路径不能包含URL方案'}, status_code=400)
-                        if '..' in idle_animation_str:
-                            return JSONResponse(content={'success': False, 'error': '待机动作路径不能包含路径遍历（..）'}, status_code=400)
+                        # 向前兼容：接受 string（旧客户端）或 list（新客户端）
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)